tox.ini

```diff
@@ -318,6 +319,8 @@ deps =
     tornado45: tornado>=4.5,<4.6
     vertica060: vertica-python>=0.6.0,<0.7.0
     vertica070: vertica-python>=0.7.0,<0.8.0
+    webapp225: webapp2>=2.5
+    webapp225: webob>=1.8,<1.9
```

Comment:
If this is really necessary, can you add a comment as why?

Reply:
Yes, apparently it is. I've added a comment and commit message with an explanation.